### PR TITLE
Algo bug fix #trivial

### PR DESCRIPTION
**Pull request by hannahli23**
Changes
----

- Fixed bug for public transit option
- #trivial
Resolved issues
-----

Closes #

### CODE REVIEW

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>The pull request description must be longer.  Include #trivial to override this.</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9f7734f4edf791c13b3a79bf05809ff93b237767
</p>

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>The pull request description must be longer.  Include #trivial to override this.</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9f7734f4edf791c13b3a79bf05809ff93b237767
</p>

File: server/routes/map.js [hunk 1]
```diff
@@ -115,7 +115,15 @@ router.get("/query", async function (req, res, next) {
             };
         });
 
-        results = results.filter(place => place.duration.value <= timeLimit);
+        results = results.filter(place =>
+        {
+            if (place.duration === undefined)
```
Comment (evan10s) [inline server/routes/map.js]: You could do this in one line with
`return place && place.duration && place.duration.value <= timeLimit`

**Review by evan10s — APPROVED** [review]

**Comment by evan10s** [comment]
If you make a commit, danger will run again and should pass

**Pull request merged**